demonstrations/tutorial_multiclass_classification.py

```diff
@@ -199,7 +207,7 @@ def accuracy(labels, hard_predictions):
 
 def load_and_process_data():
     data = np.loadtxt("multiclass_classification/iris.csv", delimiter=",")
-    X = torch.tensor(data[:, 0:feature_size])
+    X = torch.tensor(data[:, 1:feature_size+1])
```

Comment:
How come this was changed? It seems like now we'd classify features 1-4 instead of 0-3.

Reply:
This is was done by mistake as I used it to run one some dataset whose first column was just a sequence number.
```suggestion
    X = torch.tensor(data[:, 0:feature_size])
```

Reply:
Got it! Reverted this then. The reason is that the demo continues to use the Iris dataset, which could also potentially have `feature_size==5` with `data[:, 0:feature_size]`, whereas `data[:, 1:feature_size+1]` would result in an `IndexError` as the first features were skipped.